redshift_sqlalchemy/__init__.py

```diff
@@ -1,10 +1,12 @@
-from pkg_resources import get_distribution
+"""
+Compatibility module for projects referencing sqlalchemy_redshift
+by its old name "redshift_sqlalchemy".
+"""
 
-__version__ = get_distribution('sqlalchemy-redshift').version
+import sys
 
-from sqlalchemy.dialects import registry
+import sqlalchemy_redshift
 
-registry.register("redshift", "redshift_sqlalchemy.dialect", "RedshiftDialect")
-registry.register(
-    "redshift+psycopg2", "redshift_sqlalchemy.dialect", "RedshiftDialect"
-)
+
+# All references to module redshift_sqlalchemy will map to sqlalchemy_redshift
+sys.modules['redshift_sqlalchemy'] = sqlalchemy_redshift
```

Comment:
I just discovered `sys.modules`. This appears to do the magic that we want. It's possible to do imports like `from redshift_sqlalchemy.dialect import RedshiftDDLCompiler`.